src/views/AmalgamationSelection.vue

```diff
@@ -236,7 +240,8 @@ export default class AmalgamationSelection extends Vue {
         },
         amalgamationApplication: {
           nameRequest: {
-            legalType
+            legalType,
+            correctNameOption: correctNameOption
```

Comment:
OK, and please also set the initial `legalName` for short-form amalgamation. This is the legal name of the current business (which is the primary or holding business).